content/administration/supported_versions.rst

```diff
@@ -2,24 +2,28 @@
 
 .. _supported_versions:
```

Comment:
You can remove this; we should use the `doc` admonition for full page references.

Reply:
It could also be worth renaming the file with a proper redirect, for both URL consistency and content discoverability.